Tests/ChangelogProducerTests/XCTestManifests.swift

```diff
@@ -3,7 +3,7 @@ import XCTest
 #if !canImport(ObjectiveC)
 public func allTests() -> [XCTestCaseEntry] {
     return [
-        testCase(ChangelogProducerTests.allTests),
+        // testCase(ChangelogProducerTests.allTests),
```

Comment:
Why is this code commented out? Do we need it for running the test?

Reply:
We're not keeping the SPM tests up to date for now but it's good to keep it in place for if we ever like to add it back. That's why I simply commented it out for now!